snippets/base/templates/base/includes/snippet_js.html

```diff
@@ -495,34 +485,35 @@
 function sendMetric(metric, callback) {
     {# In preview mode, disable sampling, log metric, but do not send to service #}
     {% if preview %}
-    console.log("[preview mode] Sending metric: " + metric);
-    if (callback) {
-        callback();
-    }
-    return;
+      console.log("[preview mode] Sending metric: " + metric);
```

Comment:
Should we be sending metrics in preview mode? Or is this just dummy metric send posts?